src/common/libflux/info.c

```diff
@@ -62,13 +62,12 @@ char *flux_getattr (flux_t h, int rank, const char *name)
     return ret;
 }
 
-int flux_info (flux_t h, int *rankp, int *sizep, bool *treerootp)
+int flux_info (flux_t h, uint32_t *rankp, uint32_t *sizep, int *arityp)
 {
     flux_rpc_t *r = NULL;
     JSON out = NULL;
     const char *json_str;
-    int rank, size;
-    bool treeroot;
+    int arity, rank, size;
```

Comment:
Just k-ary.  We had k-nomial a while back but there really wasn't a driving use case so ti was dropped when the lua start scripts went away.